### ISSUE TITLE
Suport right click to go next month (calendar+1) in clock module?

### ISSUE THREAD

**Issue by tmpm697** [Alexays/Waybar#1545]
I'm using `waybar-git` from aur.

and want to hove over clock and do right click to see next month `calendar`

I tried:
```
"on-click-right": "<big>{:%Y %B}</big>\n<tt>{calendar+1}</tt>",
```
But seem it's not supported yet.

**Comment by LukashonakV** [comment]
Hi @tmpm697 can it be closed ?

**Comment by LukashonakV** [comment]
Hi @tmpm697 it's a good point I've already was thinking of. I think we will come up with something soon :)

**Comment by tmpm697** [comment]
I think we can just reset calendar when mouse does not hover clock if we can capture that event.

**Comment by LukashonakV** [comment]
Hi @tmpm697 , can you check the freshest revision please? As far as I see there was [PR1911](https://github.com/Alexays/Waybar/pull/1911) which implements calendar resetting on loosing focus.

**Comment by tmpm697** [comment]
yes, it worked for me, but the delay to reset calendar is a bit slow, i probably don't care about it but can we have an option for it or make it like 2-3s after loosing focus.

**Comment by LukashonakV** [comment]
Hi @tmpm697 I guess it depends on `interval` config parameter.

**Comment by tmpm697** [comment]
thanks, but what's wrong with latest commit? it seems this feature disabled by latest commit.

**Comment by LukashonakV** [comment]
Hi @tmpm697 , strange. This functionality is keep and works for me well for both modes: "year|months"

**Comment by LukashonakV** [comment]
@tmpm697 please pay attention config for Clock module is changed. See [WIKI](https://github.com/Alexays/Waybar/wiki/Module:-Clock) please. It might be root cause of your issue
